tests/Google.Ads.GoogleAds.Tests.csproj

```diff
@@ -28,7 +30,8 @@
     <EmbeddedResource Include="Util\field_mask_tests.json" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Condition="Exists('..\src\Google.Ads.GoogleAds.csproj')" Include="..\src\Google.Ads.GoogleAds.csproj" />
+    <ProjectReference Include="..\examples\Google.Ads.GoogleAds.Examples.csproj" />
+    <ProjectReference Include="..\src\Google.Ads.GoogleAds.csproj" />
```

Comment:
Did you omit the `Condition` attribute on purpose?

Reply:
I did. The tests are expected to be run directly against the project's source code (so csproj, not Nuget package. So condition checking is pointless).

I think the condition checking was a direct copy-paste a while back from the examples project.

Reply:
SGTM